presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -56,11 +56,15 @@ PrestoExchangeSource::PrestoExchangeSource(
     const folly::Uri& baseUri,
```

Comment:
Can we cover https in AnnouncerTest.cpp as well? We could do this in followup. Thanks!

Reply:
Yeah. I will change Announer and Prestoexchange tests to run in both mode. I need to refactor test code a bit to do so. Will send in the next PR.